pages/02.Documentation/10.taints-and-tolerations/default.en.md

```diff
@@ -0,0 +1,36 @@
+---
+title: 'Taints and Tolerations'
+published: true
+taxonomy:
+    tag:
+        - kubernetes
+        - gui
+---
+
+[Taints and Tolerations](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/) are used in Kubernetes to mark single nodes for specific use-cases (taints) and allowing pods to be scheduled on such nodes (tolerations).
+While it is possible to taint single nodes manually we do not recommend to do so when using your MetaKube cluster, since each update of the nodes will recreate them, and they will loose their taints. When creating a Node Deployment we provide a setting, that all nodes get created with Labels and/or Taints.
```

Comment:
manually over the Kubernetes API with kubectl

Reply:
Taints can also be set at the MachineDeployment resource over the CLI. We may want to document that.

Reply:
Fixed